gusto/timestepping/semi_implicit_quasi_newton.py

```diff
@@ -535,19 +533,24 @@ class Forcing(object):
     semi-implicit time discretisation.
     """
 
-    def __init__(self, equation, alpha):
+    def __init__(self, equation, alpha, dt=None):
         """
         Args:
             equation (:class:`PrognosticEquationSet`): the prognostic equations
                 containing the forcing terms.
-            alpha (:class:`Function`): semi-implicit off-centering factor. An
-                alpha of 0 corresponds to fully explicit, while a factor of 1
-                corresponds to fully implicit.
+           explicit_scaling (:class:`Expr`): Off-centering parameter for
```

Comment:
This docstring doesn't match with the interface, which still has an `alpha` argument.